en/pages/bank-account.md

```diff
@@ -2,33 +2,43 @@
 
 If you plan to work or live in Berlin, having a German bank account becomes mandatory. Without one you will not be able to receive your salary, rent an apartment or get a postpaid mobile phone. Fortunately, opening a bank account is usually simple and fast.
 
-Below you will find the steps required to open a Deutsche Bank (DB) and N26 bank account, but keep in mind that there are many other banks available. Choosing a bank is a very personal matter, so that decision is at your discretion.
+Below you will find the steps required to open a bank account. 
```

Comment:
I think some important details were deleted from this paragraph, like the sentence
> "Choosing a bank is a very personal matter, so that decision is at your discretion."

Reply:
Agreed. Added it back now.